src/ContaoManager/Plugin.php

```diff
@@ -11,9 +11,12 @@
 
 namespace Craffft\CssStyleSelectorBundle\ContaoManager;
 
-use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\CoreBundle\ContaoCoreBundle;
 use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
 use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use Craffft\CssStyleSelectorBundle\CraffftCssStyleSelectorBundle;
+use MadeYourDay\RockSolidCustomElements\RockSolidCustomElementsBundle;
```

Comment:
When this dependency is not existing, then an exception is thrown.

Reply:
Did you test that? In my tests it didn't. Use statements do not try to load the class.